linalg/sundials.cpp

```diff
@@ -1711,6 +1798,17 @@ void KINSolver::Mult(Vector &x,
       {
          flag = KINSetPrintLevel(sundials_mem, print_level);
          MFEM_VERIFY(flag == KIN_SUCCESS, "KINSetPrintLevel() failed!");
+
+         if (jfnk && print_level)
+         {
+            flag = SUNLinSolSetInfoFile_SPFGMR(LSA, stdout);
```

Comment:
Seems like this will lead to an error without much explanation if SUNDIALS is not built with `-DSUNDIALS_BUILD_WITH_MONITORING=ON` option.